Changelog.md

```diff
@@ -0,0 +1,50 @@
+# Changelog
+
+## [Unreleased]
+
+### Added
+
+- A dependency to the `RealModule` from the [swift-numerics package](https://github.com/apple/swift-numerics), in order to use the `Real` protocol.
+- Support for Xcode 12 beta and swift tools version 5.2
+
+### Changed
+
+- The generic `Curve` type and its properties are now constrained to types conforming to `Real`, instead of `FloatingPointMath`.
```

Comment:
```suggestion
- The generic `Curve` type and its properties are now constrained to types conforming to the `Real` protocol, instead of `FloatingPointMath`.
```